lib/generator/files/generator_cases.rb

```diff
@@ -10,16 +10,10 @@ def class_name(exercise_name_or_slug)
           filename(exercise_name_or_slug).split('_').map(&:capitalize).join
         end
 
-        def source_filepath(track_path, slug)
-          path = meta_generator_path(track_path, slug)
-          filename = filename(slug) + '.rb'
-          File.join(path, filename)
-        end
-
         private
 
         def cases_filepaths(track_path)
-          generator_glob = File.join(meta_generator_path(track_path, '*'), '*_case.rb')
+          generator_glob = File.join(track_path, 'exercises', '*', '.meta', 'generator', '*_case.rb')
```

Comment:
Inlining this falls under the 'sometimes it gets uglier on the way to getting cleaner'